test/fixtures/works.yml

```diff
@@ -0,0 +1,79 @@
+work1:
+  title: title1
+  creator: creator1
+  description: this is work1
+  publication_year: 2020
+  category: album
+  vote_count: 1
```

Comment:
Rather than adding a vote_count to the work fixture, it is best to rely on the relationship between works and votes, such that the way that a vote_count is increased is by a work having vote fixtures.